classes/Mail.php

```diff
@@ -505,7 +505,7 @@ function ($carry, $item) {
                 Context::getContext()->link = new Link();
             }
 
-            $templateVars['{shop_name}'] = Tools::safeOutput($shop->name);
```

Comment:
@PierreRambaud Couldn't store it in a variable for avoiding two call to `Configuration::get` ?

Reply:
Thanks, didn't notice we have the `Configuration::getMultiple` :)